docs/v2-transition.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to HyP3 v2
+
+As of September 30, 2021, our beta HyP3 service available at <https://hyp3.asf.alaska.edu/>
+has been retired in favor of our new On Demand service powered by HyP3 version 2 (hereafter, just "HyP3").
+
+On Demand processing through HyP3 is now available directly in [Vertex](https://search.asf.alaska.edu/){target=_blank},
+ASF's data search portal. Vertex provides a friendly interface to request 
+processing jobs and review previous jobs. To learn how to request jobs through vertex, please see:
```

Comment:
Not a big fan of this -- I really want to write *why* we retired v1 and *what benifit* the user is getting -- but I didn't like what I tried. Ideas?